garmin_txt.cc

```diff
@@ -1357,6 +1330,7 @@ garmin_txt_read()
     /* bail back to a (utf-8) char string, this format isn't worth the conversion work */
     QByteArray utf8str = buff.toUtf8();
     char* cin = csv_lineparse(utf8str.constData(), "\t", "", 0);
+    current_line_text = QString(buff).trimmed();
```

Comment:
buff is already a QString, and it was just trimmed above so
current_line_text = buff;